services/order.py

```diff
@@ -0,0 +1,33 @@
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order, Ticket, MovieSession
+
+
+@transaction.atomic
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: str = None
+) -> None:
+    order = Order.objects.create(
+        user=(get_user_model().objects.get(username=username))
```

Comment:
Consider handling the case where the user with the given `username` does not exist. You can use `get_object_or_404` or handle the `get_user_model().DoesNotExist` exception to provide a more user-friendly error message.